src/main.h

```diff
@@ -249,8 +249,8 @@ void ThreadScriptCheck();
 void PartitionCheck(bool (*initialDownloadCheck)(const CChainParams&), CCriticalSection& cs, const CBlockIndex *const &bestHeader);
 /** Check whether we are doing an initial block download (synchronizing from disk or network) */
 bool IsInitialBlockDownload(const CChainParams& chainParams);
-/** Format a string that describes several potential problems detected by the core */
-std::string GetWarnings(const std::string& strFor);
+/** Pair of timestamp and formatted string that describes several potential problems detected by the core */
+std::pair<int64_t, std::string> GetWarnings(const std::string& strFor);
```

Comment:
Can the pair be in the order (string, timestamp) so that it matches the order of arguments to `SetMiscWarning`?